server/client.go

```diff
@@ -1416,6 +1420,11 @@ func (c *client) readLoop(pre []byte) {
 				}
 				return
 			}
+			// Clear total stalled time here.
+			if c.in.tst > 0 {
+				c.rateLimitFormatWarnf("Producer was stalled for a total of %v", c.in.tst.Round(time.Millisecond))
```

Comment:
something that I noticed is that occasionally when the cluster is busy the natscli would stall for 2ms on simple queries, in case it gets noisy in the logs maybe we log when larger than 100ms?

Reply:
Stall total is now 10ms, so will use that..